### ISSUE TITLE
printf() doesn't display -32768

### ISSUE THREAD

**Issue by mike632t** [MiguelVis/mescc#1]
An integer value of -32768 behaves as I would expect but is not printed properly using printf()

Example:

```
#define INT_MIN   -32768
#include "mescc.h"
#include "printf.h"

main()
{
   int i;
   i = INT_MIN;
   printf("INT_MIN= %d\n", -32768);
   printf("INT_MIN= %d\n", INT_MIN);
   printf("INT_MIN= %d\n", i);
   printf("INT_MIN= %d\n", i + 1);
   printf("INT_MIN= %d\n", ++i);
}
```

Output:

```
INT_MIN= --.)*(
INT_MIN= --.)*(
INT_MIN= --.)*(
INT_MIN= -32767
INT_MIN= -32767
```
Thanks for making the effort to keep small-C going!

**Comment by MiguelVis** [comment]
Thanks @mike632t I will have a look.

**Comment by MiguelVis** [comment]
Hi @mike632t.

Fixed with commit a7ba53e.

```
INT_MIN= -32768
INT_MIN= -32768
INT_MIN= -32768
INT_MIN= -32767
INT_MIN= -32767
```
Thanks for opening the issue.

**Comment by mike632t** [comment]
Thanks - I'll give it a try !
